wef-subscriptions/Software-Restriction-Policies.xml

```diff
@@ -0,0 +1,30 @@
+<Subscription xmlns="http://schemas.microsoft.com/2006/03/windows/events/subscription">
+  <SubscriptionId>Software-Restriction-Policies</SubscriptionId>
+  <SubscriptionType>SourceInitiated</SubscriptionType>
+  <Description>SRP Logs from within the System log</Description>
+  <Enabled>true</Enabled>
+  <Uri>http://schemas.microsoft.com/wbem/wsman/1/windows/EventLog</Uri>
+  <ConfigurationMode>Custom</ConfigurationMode>
+  <Delivery Mode="Push">
+    <Batching>
+      <MaxItems>1</MaxItems>
+      <MaxLatencyTime>300000</MaxLatencyTime>
+    </Batching>
+    <PushSettings>
+      <Heartbeat Interval="1800000"/>
+    </PushSettings>
+  </Delivery>
+  <Query><![CDATA[
+    <QueryList>
+      <Query Id="0" Path="System">
+        <Select Path="System">*[System[Provider[@Name='Microsoft-Windows-SoftwareRestrictionPolicies'] and (EventID=865 or EventID=866 or EventID=867 or EventID=868 or EventID=882)]]</Select>

```

Comment:
If it's not too much trouble, can you add inline comments for each EventID similar to this?
 https://github.com/palantir/windows-event-forwarding/blob/ff2b753e85d3c48bcce313b445dd558be9a9984c/wef-subscriptions/Services.xml#L23-L24